.github/workflows/containerize-sample-workloads.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+name: Containerize Sample Workloads - Smart Trailer
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          install: true
+
+      - name: Login to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+  
+      - name: Build and push multi-platform Docker image for the Trailer Properties Provider
+        uses: docker/build-push-action@v5
+        with:
+          context: ./scenarios/smart_trailer
+          file: ./scenarios/smart_trailer/Dockerfile.sample_workloads
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ghcr.io/eclipse-sdv-blueprints/software-orchestration/invehicle-stack/trailer-properties-provider:0.1.0

```

Comment:
Should the version be incremented when the files are changed?

Eclipse Symphony has a release workflow (https://github.com/eclipse-symphony/symphony/blob/main/.github/workflows/release.yaml#L121) that increments their image versions.

This is the workflow step that does the incrementing https://github.com/eclipse-symphony/symphony/blob/main/.github/workflows/release.yaml#L62

Reply:
Thanks for the information and I do like the suggestion. I will consider using a similar approach when we introduce releases/ release pipelines into the blueprint-related work. I would prefer to leave this as-is for now and look to add programmatic version updates in a later change.